web/frontend/src/components/base.vue

```diff
@@ -92,19 +94,19 @@ const digestInput = () => {
   for (const item of pressed) {
     switch (item) {
       case Keymap.FORWARD: {
-        linearValue += 1;
+        linearValue += Number(0.01 * power);
```

Comment:
```suggestion
        linearValue += Number.parseFloat(0.01 * power);

Reply:
Also, just noticing, why do we need casting here at all? both power and 0.01 are numbers

Reply:
I was just limiting the number of decimal points but we don't have to.